Zotero/Scenes/Detail/ItemDetail/Views/ItemDetailTitleContentView.swift

```diff
@@ -7,52 +7,82 @@
 //
 
 import UIKit
+import RxSwift
 
 final class ItemDetailTitleContentView: UIView {
-    @IBOutlet private weak var textView: UITextView!
-    @IBOutlet private weak var topConstraint: NSLayoutConstraint!
-    @IBOutlet private weak var bottomConstraint: NSLayoutConstraint!
-    @IBOutlet private weak var separatorHeight: NSLayoutConstraint!
+    private var textView: FormattedTextView!
```

Comment:
`weak` views and constraints (unless they are being deactivated at some point)

Reply:
Check https://github.com/zotero/zotero-ios/pull/991/commits/6433d3ff8c672555585e790c4d748ca4f7cbbc6a